src/components/sound.js

```diff
@@ -64,17 +64,29 @@ module.exports.Component = registerComponent('sound', {
     }
     // All sound values set. Load in `src`.
     if (srcChanged) {
-      var self = this;
+      var isStream = (data.src.srcObject instanceof MediaStream);
+
+      if (isStream) {
+        var stream = data.src.srcObject;
+        this.pool.children.forEach(function (sound) {
+          sound.setNodeSource(sound.context.createMediaStreamSource(stream));
+        });
+        this.loaded = true;
```

Comment:
should this wait for the `loadedmetadata` event to fire?

Reply:
I assume by this point [autoplay](https://github.com/aframevr/aframe/pull/3042/files#diff-6e2259d7fcbd1d0b371631b04575bc07R90) is already assumed? what about with Notifications API prompts, etc.?

haven't tested that part - just figured I'd ask.